extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/OpenshiftConfig.java

```diff
@@ -183,6 +183,12 @@ public static enum DeploymentResourceKind {
     @ConfigItem
     Optional<List<String>> imagePullSecrets;
 
+    /**
+     * Enable generation of image pull secret, when credentials are available.
```

Comment:
```suggestion
     * Enable generation of image pull secret, when the container image username and password were provided. 
```

Or something like this.